offchain-modules/src/packages/config.ts

```diff
@@ -30,7 +30,19 @@ export interface EthConfig {
   privateKey: string;
 }
 
+export interface TronConfig {
+  privateKey: string;
+  tronGridUrl: string;
+  committee: {
+    address: string;
+    permissionId: string;
+    keys: string[];
+  };
+  startTimestamp: number;
```

Comment:
What is this used for?

Reply:
- a tron multisig address may have multiple permission groups, permissionId is used to identify permission group of a tron multisig address